config/settings.d/dns_nsupdate.yml.example

```diff
@@ -0,0 +1,16 @@
+---
+#
+# Configuration file for 'nsupdate' dns provider
+#
+# Can be true, false
+:enabled: true
+
+#:dns_key: /etc/rndc.key
+# use this setting if you are managing a dns server which is not localhost though this proxy
+#:dns_server: dns.domain.com
+# use this setting if you want to override default TTL setting (86400)
+#:dns_ttl: 86400
```

Comment:
Cool, that makes sense too.

Can you please file packaging changes required to get this working? I will do testing on live instances. Assuming there are changes necessary (the second migration script wont get triggered currently during package upgrade if I read that correctly).